src/libutil/hash.hh

```diff
@@ -20,6 +20,9 @@ const int sha1HashSize = 20;
 const int sha256HashSize = 32;
 const int sha512HashSize = 64;
 
+// BLAKE3 data size threshold beyond which parallel hashing with TBB is likely faster.
+const size_t blake3TbbThreshold = 128000;
+
```

Comment:
Does this need to be exposed in the header?

Reply:
No, it's not necessary. I can remove it from the header.

It might eventually be useful to have exposed for some other part of the code base but can always revisit that later.